ddtrace/contrib/flask/patch.py

```diff
@@ -1,11 +1,16 @@
 import json
 
 import flask
+from flask import Response
```

Comment:
nit: `flask` is already imported, you can use `flask.Response`

Reply:
True, changed.